target/linux/qualcommax/files/arch/arm64/boot/dts/qcom/ipq8072-wax620.dts

```diff
@@ -171,14 +171,15 @@
 	pinctrl-names = "default";
 	reset-gpios = <&tlmm 44 GPIO_ACTIVE_LOW>;
 
-	qca8081_28: ethernet-phy@28 {
+	qca8081: ethernet-phy@28 {
+		compatible = "ethernet-phy-id004d.d101";
```

Comment:
Commit title and description do not include this, no need to define the ID in compatible in your case as there is no reset GPIO attached.

Honestly, there isnt really a need for this commit at all as both labels are descriptive enough already

Reply:
I removed this commit.